setup.py

```diff
@@ -2,6 +2,7 @@
 import re
 from setuptools import find_packages
 from setuptools import setup
+from io import open
```

Comment:
I'd prefer `import io` and using `io.open` below, otherwise it looks like you're using the builtin `open`.